glTF-BinExporter/RhinoDocGltfConverter.cs

```diff
@@ -79,7 +83,7 @@ public Gltf ConvertToGltf()
 
             var sanitized = SanitizeRhinoObjects(objects);
 
-            foreach(ObjectExportData exportData in sanitized)
+            foreach (ObjectExportData exportData in sanitized)
             {
                 int[] materialIndices = GetMaterials(exportData.RenderMaterials, exportData.Object);
```

Comment:
The export materials check should happen here, not inside the GetMaterials function.